src/journal/models.py

```diff
@@ -1077,6 +1082,12 @@ def setup_default_form(sender, instance, created, **kwargs):
             default_review_form.elements.add(main_element)
 
 
+@receiver(post_save, sender=Journal)
+def update_issue_display_title(sender, instance, created, **kwargs):
+    for issue in Issue.objects.filter(journal=instance):
+        issue.update_display_title()
+        issue.save()
```

Comment:
It's not redundant since `update_display_title()` doesn't call `save()`. This was to avoid a circular call, since `save()` calls `update_display_title()`.

Or is django smart enough to avoid a loop between the two functions?

Reply:
Ah, sorry, misread your earlier comment. Will remove the call to update.